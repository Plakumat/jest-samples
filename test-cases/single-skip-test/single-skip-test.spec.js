import movies from "./single-skip-test"

describe('Favorite Movies', () => {
    let myMovies = []
    beforeEach(() => {
        myMovies = [{
        title: 'Terminator',
        rate: null
        }]
    })

    //Jest will test only this case
    test.only('can add a movie', () => {
        movies.add(myMovies, 'John Wick')
        expect(myMovies).toMatchSnapshot()
    })

    //Jest will skip this case
    test.skip('rate a movie', () => {
        movies.rate(myMovies[0], 5)
        expect(myMovies).toMatchSnapshot()
    })
})