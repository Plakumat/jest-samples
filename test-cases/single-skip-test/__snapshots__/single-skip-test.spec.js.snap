// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite Movies can add a movie 1`] = `
Array [
  Object {
    "rate": null,
    "title": "Terminator",
  },
  Object {
    "rate": null,
    "title": "John Wick",
  },
]
`;
